validation/open_boundaries/cylinder.jl

```diff
@@ -18,10 +18,10 @@ using Oceananigans.BoundaryConditions: FlatExtrapolationOpenBoundaryCondition,
 Returns the drag within the `bounding_box` computed by:
 
 ∂ₜu⃗ + (u⃗⋅∇)u⃗ = −∇P + ∇⋅τ⃗ + F⃗
-F⃗ₜ =∫ᵥF⃗dV = ∫ᵥ(∂ₜ u⃗ + (u⃗⋅∇)u⃗ + ∇P − ∇⋅τ⃗)dV
-F⃗ₜ=∫ᵥ(∂ₜu⃗)dV + ∮ₛ(u⃗(u⃗⋅n̂) + Pn̂ − τ⃗⋅n̂)dS
-Fᵤ=∫ᵥ ∂ₜu dV + ∮ₛ(u(u⃗⋅n̂) − τₓₓ)dS + ∮ₛPx̂⋅dS⃗
-Fᵤ=∫ᵥ ∂ₜ u dV − ∫ₛ₁(u² − 2ν∂ₓ u + P)dS + ∫ₛ₂(u² − 2ν∂ₓ u+P)dS − ∫ₛ₃uvdS + ∫ₛ₄ uvdS
+F⃗ₜ = ∫ᵥF⃗dV = ∫ᵥ(∂ₜ u⃗ + (u⃗⋅∇)u⃗ + ∇P − ∇⋅τ⃗)dV
+F⃗ₜ = ∫ᵥ(∂ₜu⃗)dV + ∮ₛ(u⃗(u⃗⋅n̂) + Pn̂ − τ⃗⋅n̂)dS
+Fᵤ = ∫ᵥ ∂ₜu dV + ∮ₛ(u(u⃗⋅n̂) − τₓₓ)dS + ∮ₛPx̂⋅dS⃗
+Fᵤ = ∫ᵥ ∂ₜ u dV − ∫ₛ₁(u² − 2ν∂ₓ u + P)dS + ∫ₛ₂(u² − 2ν∂ₓ u+P)dS − ∫ₛ₃uvdS + ∫ₛ₄ uvdS
```

Comment:
FYI what this looks like on my screen / browser

<img width="810" alt="image" src="https://github.com/user-attachments/assets/98dd54f5-59ce-436b-b735-73e2ffe227d9" />

Reply:
I looks similarly messed up on my browser, but the actual document looks fine on my laptop:

![image](https://github.com/user-attachments/assets/9732ae2a-f596-4e20-bf81-84197b171f5a)

Probably it's best to prioritize rendering on github, since a lot of people use it. I guess the easiest way to do that is to copy-paste stuff on a github text box and see what displays correctly and what doesn't. I just tested it out right now and it seems to work. The `\oint` character seems to not work for example.

Reply:
> Probably it's best to prioritize rendering on github, since a lot of people use it. I guess the easiest way to do that is to copy-paste stuff on a github text box and see what displays correctly and what doesn't. I just tested it out right now and it seems to work. The `\oint` character seems to not work for example.

I tried this but then when I copy it over it renders wrong:

```math
\frac{\partial \vec{u}}{\partial t} + (\vec{u}\cdot\nabla)\vec{u}=-\nabla P + \nabla\cdot\vec{\tau} + \vec{F},
```
```math
\vec{F}_T=\int_\Omega\vec{F}dV = \int_\Omega\left(\frac{\partial \vec{u}}{\partial t} + (\vec{u}\cdot\nabla)\vec{u}+\nabla P - \nabla\cdot\vec{\tau}\right)dV,\newline
```
```math
\vec{F}_T=\int_\Omega\left(\frac{\partial \vec{u}}{\partial t}\right)dV + \oint_{\partial\Omega}\left(\vec{u}(\vec{u}\cdot\hat{n}) + P\hat{n} - \vec{\tau}\cdot\hat{n}\right)dS,
```

```math
F_u=\int_\Omega\left(\frac{\partial u}{\partial t}\right)dV + \oint_{\partial\Omega}\left(u(\vec{u}\cdot\hat{n}) - \tau_{xx}\right)dS + \int_{\partial\Omega}P\hat{x}\cdot d\vec{S},
```

```math
F_u=\int_\Omega\left(\frac{\partial u}{\partial t}\right)dV - \int_{\partial\Omega_1} \left(u^2 - 2\nu\frac{\partial u}{\partial x} + P\right)dS + \int_{\partial\Omega_2}\left(u^2 - 2\nu\frac{\partial u}{\partial x}+P\right)dS -  \int_{\partial\Omega_2} uvdS + \int_{\partial\Omega_4} uvdS
```

where the bounding box is $\Omega$ which is formed from the boundaries $\partial\Omega_1$, $\partial\Omega_2$, $\partial\Omega_3$, $\partial\Omega_4$ which have outward directed normals $-\hat{x}$, $\hat{x}$, $-\hat{y}$ and $\hat{y}$

Rendered in julia file:
∂ u → ∂ t + ( u → ⋅ ∇ ) u → = − ∇ P + ∇ ⋅ τ → + F → ,
F → T = ∫ Ω F → d V = ∫ Ω ( ∂ u → ∂ t + ( u → ⋅ ∇ ) u → + ∇ P − ∇ ⋅ τ → ) d V ,
F → T = ∫ Ω ( ∂ u → ∂ t ) d V + ∮ ∂ Ω ( u → ( u → ⋅ n ^ ) + P n ^ − τ → ⋅ n ^ ) d S ,
F u = ∫ Ω ( ∂ u ∂ t ) d V + ∮ ∂ Ω ( u ( u → ⋅ n ^ ) − τ x x ) d S + ∫ ∂ Ω P x ^ ⋅ d S → ,
F u = ∫ Ω ( ∂ u ∂ t ) d V − ∫ ∂ Ω 1 ( u 2 − 2 ν ∂ u ∂ x + P ) d S + ∫ ∂ Ω 2 ( u 2 − 2 ν ∂ u ∂ x + P ) d S − ∫ ∂ Ω 2 u v d S + ∫ ∂ Ω 4 u v d S

where the bounding box is Ω which is formed from the boundaries ∂ Ω 1 , ∂ Ω 2 , ∂ Ω 3 , ∂ Ω 4 which have outward directed normals − x ^ , x ^ , − y ^ and y

I'll play around and try and find an intermediary

Reply:
How does this look:
```suggestion
∂ₜU + (U⋅∇)U = −∇P + ∇⋅τ + F⃗
F = ∫ᵥFdV = ∫ᵥ(∂U/∂t + (U⋅∇)U + ∇P − ∇⋅τ)dV
F = ∫ᵥ(∂U/∂t)dV + ∫ₛ(U(U⋅n̂) + Pn̂ − τ⋅n̂)dS
Fᵤ = ∫ᵥ ∂U/∂t dV + ∫ₛ(u(U⋅n̂) − τₓₓ)dS + ∮ₛPx̂⋅dS⃗
Fᵤ = ∫ᵥ ∂U/∂t dV − ∫ₛ₁(u² − 2ν∂ₓ u + P)dS + ∫ₛ₂(u² − 2ν∂ₓ u+P)dS − ∫ₛ₃uvdS + ∫ₛ₄ uvdS
```

I think I got all of the unrendered characters gone (vectors and oint?).

Reply:
Strange that GH renders the original fine for me, maybe because I have a systemwide juliamono installation?

Reply:
not sure. But your update renders fine.

Reply:
It renders almost 100% fine for me, except for a couple of characters:

![image](https://github.com/user-attachments/assets/6cb81488-f325-48b4-b13a-7a7659aaef98)

Reply:
oh yeah i missed that. I also see boxes for that term.

Reply:
How does this look:
```suggestion
∂ₜU + (U⋅∇)U = −∇P + ∇⋅τ + F
F = ∫ᵥFdV = ∫ᵥ(∂U/∂t + (U⋅∇)U + ∇P − ∇⋅τ)dV
F = ∫ᵥ(∂U/∂t)dV + ∫ₛ(U(U⋅n̂) + Pn̂ − τ⋅n̂)dS
Fᵤ = ∫ᵥ ∂U/∂t dV + ∫ₛ(u(U⋅n̂) − τₓₓ)dS + ∫ₛPx̂⋅dS⃗
Fᵤ = ∫ᵥ ∂U/∂t dV − ∫ₛ₁(u² − 2ν∂ₓ u + P)dS + ∫ₛ₂(u² − 2ν∂ₓ u+P)dS − ∫ₛ₃uvdS + ∫ₛ₄ uvdS
```
how about now?

Reply:
Almost 
![image](https://github.com/user-attachments/assets/cbde9065-e699-45cf-a3be-6cd31cdc856e)

Reply:
Also for the record, I just set juliamono as my browser font and things display correctly. So this appears to be system-dependent. Given that, should we still try to tailor the characters to appear on github? Whether or not it displays is apparently dependent on the browser, rather than on github...

Reply:
fair point

Reply:
How about now:
```suggestion
∂ₜU + (U⋅∇)U = −∇P + ∇⋅τ + F
F = ∫ᵥFdV = ∫ᵥ(∂U/∂t + (U⋅∇)U + ∇P − ∇⋅τ)dV
F = ∫ᵥ(∂U/∂t)dV + ∫ₛ(U(U⋅n̂) + Pn̂ − τ⋅n̂)dS
Fᵤ = ∫ᵥ ∂U/∂t dV + ∫ₛ(u(U⋅n̂) − τₓₓ)dS + ∫ₛPx̂⋅dS
Fᵤ = ∫ᵥ ∂U/∂t dV − ∫ₛ₁(u² − 2ν∂ₓ u + P)dS + ∫ₛ₂(u² − 2ν∂ₓ u+P)dS − ∫ₛ₃uvdS + ∫ₛ₄ uvdS
```

Reply:
Seems to work now (on Fira Code, which I think is the default for Chrome)! Thanks for the patience @jagoosw. Let's resolve this issue and move on :)